app/helpers/report_helper/report_information_helper.rb

```diff
@@ -0,0 +1,108 @@
+module ReportHelper::ReportInformationHelper
+  def report_role(role)
+    role_allows?(:feature => role)
+  end
+
+  def report_dictionary(text, type)
+    Dictionary.gettext(text, :type => type, :notfound => :titleize)
+  end
+
+  def on_click(type, accord)
+    "miqJqueryRequest('/report/x_show/#{type}?accord=#{accord}');"
+  end
+
+  def report_information(sb_data, report)
+    if sb_data[:miq_report_id] && report
+      data = summary_data(_('Basic Information'))
+      rows = []
+      rows.push(row_data(_('ID'), report.id))
+      rows.push(row_data(_('Title'), report.title))
+      rows.push(row_data(_('Primary (Record) Filter'), report.conditions.to_human)) if report.conditions
+      rows.push(row_data(_('Secondary (Display) Filter'), report.display_filter.to_human)) if report.display_filter
+      if report.sortby
+        sortby = []
+        report.sortby.each do |s|
+          sortby.push(report_dictionary(s, :column))
+        end
```

Comment:
_NOT_ for this PR, since this is a straight conversion PR, but there are a lot of opportunities here for cleaner Ruby code now that it's not in haml. For example,

```suggestion
        sortby = report.sortby.map { |s| report_dictionary(s, :column) }
```